portfolio/src/main/webapp/script.js

```diff
@@ -104,3 +104,10 @@ window.onclick = function(event) {
   if (event.target === viewModal) viewModal.style.display = "none";  
   if (event.target === connectModal) connectModal.style.display = "none";
 }
+
+function createMap() {
+  const map = new google.maps.Map(
```

Comment:
rename `map` to be more descriptive. what is it a map of? hard to tell based on coordinates and zoom.